src/client/pages/PackagesPage.tsx

```diff
@@ -218,7 +213,7 @@ const PackagesPage: React.FC<PackagesPageProps> = ({ loginCallback = '/', items,
             </CreatePackageModalLink>
           </Flexbox>
         )}
-        {items.length === 0 ? (
+        {packages.length === 0 ? (
```

Comment:
This fixes issue 1, as only the original items array was checked to decide to render packages or not